tests/test_multidb.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase
+
+from admin_interface.models import Theme
+
+from .routers import DatabaseAppsRouter
+
+
+class AdminInterfaceModelsWithDBRoutingTestCase(TestCase):
+    databases = ["replica"]
+
+    def test_standard_dbrouter(self):
+        router = DatabaseAppsRouter()
+        db_for_theme = router.db_for_read(Theme)
+        assert db_for_theme == "default"
+
+    def test_dbrouter_selects_correct_db(self):
+        DATABASE_APPS_MAPPING = {
+            "admin_interface": "replica",
+        }
+        router = DatabaseAppsRouter(db_map=DATABASE_APPS_MAPPING)
+        db_for_theme = router.db_for_read(Theme)
+        assert db_for_theme == "replica"
+
+    def test_dbrouter_errors_when_fetching_from_default(self):
+        self.assertRaises(Exception, Theme.get_active_theme)
```

Comment:
`Theme.get_active_theme` is a method but here the method is not called (missing `()`).

Reply:
No, assertRaises calls the callable you pass to it (otherwise an exception would be raised and the rest of the line would be skipped!)

There is another way to use it that people may find clearer: ```
with self.assertRaises(Exception):
    Theme.get_active_theme()
```

Reply:
Ahh True, that's how I generally use it!

Reply:
Updated to use with `with`